src/test/java/com/purbon/kafka/topology/integration/containerutils/SslKsqlContainer.java

```diff
@@ -5,8 +5,7 @@
 public class SslKsqlContainer extends KsqlContainer {
   public SslKsqlContainer(AlternativeKafkaContainer kafka, String truststore, String keystore) {
     super(kafka);
-    this.withEnv("KSQL_LISTENERS", "https://0.0.0.0:8088")
-        .withEnv("KSQL_SSL_TRUSTSTORE_LOCATION", "/etc/ksql/secrets/truststore.jks")
+    this.withEnv("KSQL_SSL_TRUSTSTORE_LOCATION", "/etc/ksql/secrets/truststore.jks")
```

Comment:
This does not look correct.

Reply:
I was blind. Didn't notice `https` vs `http` in the super class. :-(